wiki/en/Running-a-Server.md

```diff
@@ -19,87 +19,67 @@ permalink: "/wiki/Running-a-Server"
 
 
 
-## Do I need to run a server?
+## Do I need to run a Server?
 
-The short answer is **no**. For various reasons, you should try to use other people's servers, at least at first while getting your sound set up with the Jamulus client.
+The short answer is **no**. For various reasons, you should try to use other people's Servers, at least at first while getting your sound set up with the Jamulus client.
 
-Note that you can have a "private" session with other people on a Public Server by simply soloing each other. You will then not be able to hear anyone else if they enter your server.
+Note that you can have a "private" session with other people on a Server by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from using the chat function, or seeing your profile information).
 
-If you do decide to run a server, please read the following. Setting up a server is easy, however it is better to know the theory first:
+If you do decide to run a Server, please read the following. Setting up a Server is easy, however it is better to know the theory first:
 
```

Comment:
I'd add a note about third party hosting, as well, before going onto how to do it yourself.

Reply:
What do you mean by third party?

Reply:
Currently, we talk about using other people's public servers and running your own server.  We don't mention hosting solutions like melomax.live or koord, which might be more useful to many people than having to set up their own cloud-based service.

Reply:
Ah OK. Maybe if we just say there are third party hosting options. If people want to know the details they'll probably Google for them (or post on forums)

Reply:
Yep.  Maybe have something along these lines...

- Think whether you need your own server - can you use the existing, publicly registered ones
- Think whether you want to run your own server or use a third party Jamulus-specific hosting provider
- Think whether you want to run on your own LAN or a generic cloud hosting provider

And then go into how to do it yourself in the various ways.